Polyfills/Canvas/Source/Colors.h

```diff
@@ -212,19 +212,19 @@ namespace Babylon::Polyfills::Internal
             }
             
             // matches strings of the form rgb(#,#,#) or rgba(#,#,#,#)
-            static const std::regex rgbRegex("rgba?\\(\\s*(\\d{1,3})\\s*,\\s*(\\d{1,3})\\s*,\\s*(\\d{1,3})\\s*(?:,\\s*(\\d{1,3}))?\\s*\\)");
+            static const std::regex rgbRegex("rgba?\\(\\s*(-?\\d{1,3})\\s*,\\s*(-?\\d{1,3})\\s*,\\s*(-?\\d{1,3})\\s*(?:,\\s*(-?\\d{1,3}))?\\s*\\)");
             std::smatch rgbMatch;
             if (std::regex_match(str, rgbMatch, rgbRegex))
             {
                 if (rgbMatch.size() == 5)
                 { 
                     if (rgbMatch[4].matched)
                     {
-                        return nvgRGBA(std::stoi(rgbMatch[1]), std::stoi(rgbMatch[2]), std::stoi(rgbMatch[3]), std::stoi(rgbMatch[4]));
+                        return nvgRGBA(std::clamp(std::stoi(rgbMatch[1]), 0, 255), std::clamp(std::stoi(rgbMatch[2]), 0, 255), std::clamp(std::stoi(rgbMatch[3]), 0, 255), std::clamp(std::stoi(rgbMatch[4]), 0, 255));
```

Comment:
Kind of tempted to put the limits (0 and 255) in constants to prevent any types, but then that's also kind of the same as just using `std::numeric_limits<uint8_t>::min()` and `std::numeric_limits<uint8_t>::max()`. Meh up to you.

Reply:
Ya I wondered about this, too, and decided 0 and 255 was more readable.